cmd/ingress-operator/serve-healthcheck.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
```

Comment:
The cobra import should be in a separate group between the first group of imports and the last group.